async/async-and-await/vb/readme.md

```diff
@@ -1,15 +1,15 @@
 ---
 languages:
 - vb
- products:
+products:
 - dotnet-core
 - windows
 page_type: sample
 name: "Async Sample: Asynchronous Programming with Async and Await"
 urlFragment: "async-and-await-vb"
 description: "A .NET Core WPF application that contains the example method from Asynchronous Progamming with Async and Await in Visual Basic."
 ---
-# Introduction
+# Async Sample: Asynchronous Programming with Async and Await in Visual Basic
```

Comment:
I'm not sure why this heading was changed in #1773. This heading is at the very top and it will look strange to be "Introduction"

Compare both:

![image](https://user-images.githubusercontent.com/31348972/69475105-a1acbe00-0dd1-11ea-82af-832fbde3827d.png)

***

![image](https://user-images.githubusercontent.com/31348972/69475111-cbfe7b80-0dd1-11ea-8643-12d86d4f7e54.png)


***

If you would like to include an "Introduction" heading. It will be better to be an H2.